pallets/apollo-platform/src/lib.rs

```diff
@@ -1325,6 +1455,22 @@ pub mod pallet {
             )
         }
 
+        fn on_runtime_upgrade() -> Weight {
```

Comment:
Please move runtime upgrade to empty struct with `OnRuntimeUpgrade` implementation and add it to runtime/src/migrations.rs. [Example](https://github.com/sora-xor/sora2-network/blob/master/pallets/multicollateral-bonding-curve-pool/src/migrations/v4.rs)